utilities/BuildUtilities.groovy

```diff
@@ -558,6 +570,9 @@ def getDeployType(String langQualifier, String buildFile, LogicalFile logicalFil
 			} else if (isDLI(logicalFile)){
 				String dliDeployType = props.getFileProperty("${langQualifier}_deployTypeDLI", buildFile)
 				if (dliDeployType != null) deployType = dliDeployType
+			} if (isIMS(logicalFile)){
```

Comment:
Is there an 'else' missing ?
And what happens for IMS transactions that do IMS DL/I, they are flagged as `deployTypeIMS`? Is it correct?
In the end, `deployTypeIMS` and `deployTypeDLI` are all pointing to `IMSLOAD`, so is there a need to distinguish `deployTypeIMS` and `deployTypeDLI`? I don't know the answer, just asking :)

Reply:
@M-DLB Good catch, it is indeed cleaner with the `else if`. - Added with the last commit.

To your comment about the deployType for EXEC DLI - 

I intentionally changed the order for the deploy types, because they indicate the runtime environment to the deployment solution.

While EXEC DLI can be used for multiple runtimes, the `isCICS`, and `isIMS` flag are considered before the `isDLI` flag. See - https://www.ibm.com/docs/en/ims/15.4.0?topic=execution-translator-compiler-binder-options-required-exec-dli

It might need some refactoring to reevaluate the isDLI flag ...